Node.gitignore

```diff
@@ -3,6 +3,10 @@ logs
 *.log
 npm-debug.log*
 
+# .DS_Store
```

Comment:
This is an OSX specific rule and tracked in `Global/OSX.gitignore` - if you think that rule needs to be changed please open a PR against that file.